resources/helm/v2.3/istio-telemetry/grafana/templates/pvc.yaml

```diff
@@ -14,5 +14,5 @@ spec:
     - {{ .Values.grafana.accessMode }}
   resources:
     requests:
-      storage: 5Gi
+      storage: {{ .Values.grafana.persistenceResources.requests.storage | quote }}
```

Comment:
You should apply this change to a corresponding file in the directory `helm/overlays` and then run `make gen`.

Reply:
As discussed with @luksa, I do the charts changes using the script [patch-charts](https://github.com/maistra/istio-operator/blob/maistra-2.3/build/patch-charts.sh)